packages/examples/main/src/langium/statemachineClient.ts

```diff
@@ -146,7 +145,6 @@ try {
         debugLogging: true,
         logLevel: LogLevel.Info
     });
-    // renderPanelPart(document.querySelector<HTMLDivElement>('#panel')!);
```

Comment:
Do you want to keep the comments?

Reply:
No, I removed them for now. Thinking about actually adding another example that could make use of views (e.g. #521)